examples/goldenspike_examples/goldenspike.yml

```diff
@@ -18,19 +18,19 @@ stages:
   name: flow_creator_train
   nprocess: 1
 - classname: LSSTErrorModel
-  module_name: rail.creation.degradation.lsst_error_model
+  module_name: rail.creation.degraders.lsst_error_model
   name: lsst_error_model_train
   nprocess: 1
 - classname: InvRedshiftIncompleteness
-  module_name: rail.creation.degradation.spectroscopic_degraders
+  module_name: rail.creation.degraders.spectroscopic_degraders
   name: inv_redshift
   nprocess: 1
 - classname: LineConfusion
-  module_name: rail.creation.degradation.spectroscopic_degraders
+  module_name: rail.creation.degraders.spectroscopic_degraders
   name: line_confusion
   nprocess: 1
 - classname: QuantityCut
-  module_name: rail.creation.degradation.quantityCut
+  module_name: rail.creation.degraders.quantityCut
   name: quantity_cut
   nprocess: 1
 - classname: ColumnMapper
```

Comment:
Looking at the reference to some rail.core.utilStages here, I wonder if we shouldn't just remove this file completely. When I try to search "utilStages" in my all-rail directory, nothing shows up.

I'm worried this file is probably a little old and unmaintained, and might be more harmful than helpful if we keep it around in a broken state. Additionally, I've found that recent opinions seem to be that any yaml pipeline files will live either in rail_pipelines or some other repo designed explicitly for storing and sharing pipelines.

Reply:
(And when I say here, I mean a couple lines below, in the Columm Mapper stage and a few others. Github inconveniently limits comments to code the PR touches, which, I guess, fair.)

Reply:
Thanks, yeah I get that, it is my negligence for missing the `core.utilStages` here, they should be `tools.table_tools`. I will fix it for now but let's discuss during the tag-up about whether we still want yml in the examples here .